packages/p2p/src/components/orders/order-details/order-actions-block.jsx

```diff
@@ -34,11 +36,34 @@ const OrderActionsBlock = ({ cancelPopup, order_details, showPopup }) => {
             title: localize('Cancel this order?'),
             message: localize('If you have paid, please do not cancel the order.'),
             confirm_text: localize('Cancel this order'),
+            has_cancel: true,
+            cancel_text: localize('Do not Cancel'),
             onClickConfirm: cancel,
         };
         showPopup(options);
     };
-
+    const compalin = () => {
+        const options = {
+            title: localize('Something went wrong?'),
```

Comment:
Has this been checked by copywriting? Question mark seems wrong.

Reply:
it is based on Zeplin design 
https://app.zeplin.io/project/5d0b634adaf19b70cdccb972/screen/5e4baf4622aeb4a9f7d581a9

Reply:
updated